tests/query_runtime_errors/bad_state_window_return/error.txt

```diff
@@ -0,0 +1 @@
+Error: Window returned `"snot"` this can't be used as a window reply
```

Comment:
The error would be more helpful if it was describing what can be used as a window reply.

```suggestion
Error: Window returned `"snot"` this can't be used as a window reply
```